readme.md

```diff
@@ -2,45 +2,66 @@
 
 [<img src="https://babeljs.io/images/logo.svg" width="160" align="right" alt="babel">](http://babel.io)
 
-> A list of awesome Babel plugins, presets, etc.
+> A list of awesome Babel plugins, presets, etc. Many of these are from the community, but some are lesser-known
+> plugins in the Babel organization that may be useful to you.
 
 If you want to contribute, please read the [contribution guidelines](contributing.md).
 
-## Table of Contents
-
-- [Parsers](#parsers)
-- [Plugins](#plugins)
- - [Frameworks and Libraries](#frameworks-and-libraries)
- - [Testing](#testing)
-- [Presets](#presets)
-
 ## Parsers
 
  - [babel-eslint](https://github.com/babel/babel-eslint) - ESLint using Babel as the parser.
+   - Note: ESLint now lints most ES6+ syntax. This parser is only necessary if you are using Flow types or other experimental features.
```

Comment:
Thinking about it now, it would be nice if the eslint unexpected token error message said something like if you are using experimental syntax use babel-eslint or something.